cmd/root.go

```diff
@@ -82,6 +82,9 @@ func init() {
 	// Cobra also supports local flags, which will only run
 	// when this action is called directly.
 	// RootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
+	if os.Getenv("NODE_NAME") != "" {
```

Comment:
to be clear, if --nodename is specified, the value will override the NODE_NAME environment variables, right?

Reply:
In this case, shall we make it the "defaultNodeName"?
move it before the PersistentFlags sections, and change the envVar name to DEFAULT_NODE_NAME

```
defaultNodeName := os.Getenv("DEFAULT_NODE_NAME")
if  defaultNodeName == "" {
    defaultNodeName = "virtual-kubelet"
}
...
RootCmd.PersistentFlags().StringVar(&nodeName, "nodename", defaultNodeName, "kubernetes node name")

```